keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -614,6 +635,43 @@ Go in to as much detail as necessary here.
 This might be a good place to talk about core concepts and how they relate.
 -->
 
+#### Failing delete after a condition is added
+
+Here we consider a scenario when a component fails (for example its container
+dies) between appending a pod condition and deleting the pod.
+
+In particular, in case of Scheduler it is possible that it decides to preempt
+a different pod the next time (or none). Then, this pod would be left with a
+condition that the pod was preempted, when it actually wasn't. This in turn
+could lead to inproper handling of the pod by the job controller.
+
+One possible solution to this issue would be to introduce a controller in
+kube-controller-manager that either completes a preemption (does a delete)
+or removes the condition (or set it's status to false) if the condition
+timestamp is old enough and there is no deletion timestamp.
+
+We are going to re-evaluate the need for solving this issue and the solution
+for Beta.
+
+#### Marking pods as Failed
+
+As indicated in by our experiments (see: [Disconnected node](#disconnected-node)
+and [Disconnected node when taint-manager is disabled](#disconnected-node-when-taint-manager-is-disabled))
```

Comment:
We are getting rid of the ability to disable taint-manager, so this one is no longer a concern.

Reply:
Updated this section. Still it seems to me beneficial to leave the experiment results for this scenario to demonstrate we gave some thought to it.